delphi/GrFN/networks.py

```diff
@@ -462,20 +473,18 @@ def from_python_src(
 
         G = cls.from_dict(pgm_dict, lambdas_path)
 
-        if not save_intermediate_files:
-            # Cleanup intermediate files.
-            variable_map_filename = python_file.replace(".py", "_variable_map.pkl")
-            rectified_xml_filename = "rectified_" + str(Path(python_file)).replace(
-                ".py", ".xml"
-            )
-            os.remove(variable_map_filename)
-            os.remove(rectified_xml_filename)
-
+        # Cleanup intermediate files.
+        variable_map_filename = python_file.replace(".py", "_variable_map.pkl")
+        os.remove(variable_map_filename)
+        rectified_xml_filename = "rectified_" + str(Path(python_file)).replace(
+            ".py", ".xml"
+        )
+        os.remove(rectified_xml_filename)
         return G
 
     @classmethod
     def from_fortran_file(
-        cls, fortran_file: str, tmpdir: str = ".", save_intermediate_files: bool = False
+        cls, fortran_file: str, tmpdir: str = ".", save_file: bool = False
```

Comment:
Same issue here - save_file is not used elsewhere in the code so we might want to remove it...